test/e2e/v4/plugin_cluster_test.go

```diff
@@ -112,43 +112,43 @@ func Run(kbc *utils.TestContext, hasWebhook, isToUseInstaller, isToUseHelmChart,
 
 	By("creating manager namespace")
```

Comment:
Hi @Sijoma 

Please, ensure that you have your changes released with master
Also, please squash the commits; We should not have more than 1 commit to be able to get it merged

Thank you 🥇

Reply:
Squashed & Rebased it now. 

Thanks!